### PR TITLE
fix(docs): made the docs consistent for all frameworks similar to astro

### PR DESCRIPTION
**Pull request by Shubhdeep12**
Before opening this PR:

- [ ] I added a [Changeset Entry](https://spotlightjs.com/contribute/changesets/) with `pnpm changeset:add`
- [x] I referenced issues that this PR addresses

#370

### CODE REVIEW

File: packages/website/src/content/docs/setup/other.mdx [hunk 1]
```diff
@@ -116,7 +118,7 @@ Spotlight's [architecture](/architecture/) requires a sidecar to be running alon
 </CardGrid>
 
 
-## Leverage Sentry SDKs
+## Bonus: Leverage Sentry SDKs
 
 To get the most value out of Spotlight, configure Sentry's SDKs. You don't need to be using the Sentry service to take advantage of Sentry's SDKs to power Spotlight.
```
Comment (Lms24) [inline packages/website/src/content/docs/setup/other.mdx:123]:
```suggestion
To get the most value out of Spotlight, configure Sentry's SDKs. You don't have to use sentry.io but you can use the them to just power Spotlight.
```

File: packages/website/src/content/docs/setup/sveltekit.mdx [hunk 2]
```diff
@@ -9,16 +9,21 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 This guide gets Spotlight up and running in SvelteKit with Sentry. 
 You don't strictly need Sentry for Spotlight to work, we highly recommend it.
```
Comment (Lms24) [inline packages/website/src/content/docs/setup/sveltekit.mdx:10]:
```suggestion
You don't strictly need Sentry for Spotlight to work but we highly recommend it.
```

File: packages/website/src/content/docs/setup/sveltekit.mdx [hunk 3]
```diff
@@ -9,16 +9,21 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
...
 
-## Install Sentry 
+## 1. Installation
+
+#### Install Sentry
```
Comment (Lms24) [inline packages/website/src/content/docs/setup/sveltekit.mdx:14]: Is there a reason we're skipping the h3 level (`###`) here?

File: packages/website/src/content/docs/setup/sveltekit.mdx [hunk 4]
```diff
@@ -9,16 +9,21 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
...
 
 Follow the [Sveltekit Getting Started](https://docs.sentry.io/platforms/javascript/guides/sveltekit/) guide on Sentry to install and configure Sentry.
```
Comment (Lms24) [inline packages/website/src/content/docs/setup/sveltekit.mdx:16]:
```suggestion
Follow the [Sveltekit Getting Started](https://docs.sentry.io/platforms/javascript/guides/sveltekit/) guide to install and configure Sentry.
```

File: packages/website/src/content/docs/setup/sveltekit.mdx (hunk 3 above)
Comment (Shubhdeep12) [reply packages/website/src/content/docs/setup/sveltekit.mdx:14]: No, there isn't a specific reason for choosing h4 @Lms24 
The visual difference in sizes between h3 and h2 was minimal, making it difficult to distinguish on the UI, i.e. why I opted for h4 to ensure clearer visual hierarchy.

Comment (Lms24) [reply packages/website/src/content/docs/setup/sveltekit.mdx:14]: Ah I see, yeah no worries then let's leave it as is. Maybe we should make the difference between h2 and h3 stronger in general but let's not do this in this PR.

File: packages/website/src/content/docs/setup/other.mdx [hunk 5]
```diff
@@ -104,7 +106,7 @@ Sentry.init({
 
 ---
 
-## Run the Sidecar
+## 3. Run the Sidecar
```
Comment (Lms24) [inline packages/website/src/content/docs/setup/other.mdx:109]: Just noticed (can't mark the line though in the GH UI): Can we remove the `:::tip` block in lines 91-103? I think this is essentially a duplicate of the newer "Leverage Sentry SDKs" section

Comment (Shubhdeep12) [reply packages/website/src/content/docs/setup/other.mdx:94]: Done, removed. the Leverage Sentry SDK section is fine. no need of this tip.

**Review by Lms24 — APPROVED** [review]
thanks!

**Pull request merged**